src/cmds/msleep/Mybuild

```diff
@@ -2,21 +2,21 @@
 package embox.cmd
 
 @AutoCmd
-@Cmd(name = "sleep",
-	help = "sleep for specified usec num",
+@Cmd(name = "msleep",
+	help = "msleep for specified msec num",
 	man = '''
 		NAME
-			sleep - freezes for specified number of useconds.
+			msleep - freezes for specified number of mseconds.
 		SYNOPSIS
-			sleep USEC
+			msleep MSEC
 		DESCRIPTION
 
 		AUTHOR
 			Anton Kozlov
 	''')
-module sleep {
+module msleep {
 	@IncludePath("$(SRC_DIR)/compat/posix/include")
-	source "sleep.c"
+	source "msleep.c"
```

Comment:
msleep.c have usage(), that still have "USEC"

Reply:
@AntonKozlov Thanks. Fixed it. Look up again, please.